tests/unit/StoreActions.spec.ts

```diff
@@ -63,7 +63,6 @@ describe('Business Actions', () => {
     await rootStore.loadStateFiling()
     expect(LegalServices.fetchFiling).toHaveBeenCalled()
     expect(rootStore.stateFiling.business.identifier).toBe('BC0871273')
-    expect(rootStore.stateFiling.consentContinuationOut.expiry).toBe('2023-12-31')
```

Comment:
There is no `consentContinuationOut` field in `stateFiling`. Eslint was complaining about this.

Reply:
Actually, it looks like this unit test caught an error -- `StateFilingIF` should have a "consentContinuationOut" object (just as it can have a dissolution, resolution, etc objects).

Line 51 above shows the object.

Also please restore this test :)

Reply:
Fixed, thanks.